HopsanGUI/LogDataHandler2.h

```diff
@@ -72,6 +72,7 @@ class LogDataHandler2 : public QObject
     void exportToCSV(const QString &rFilePath, const QList<SharedVectorVariableT> &rVariables) const;
     void exportGenerationToCSV(const QString &rFilePath, int gen) const;
     void exportToHDF5(const QString &rFilePath, const QList<SharedVectorVariableT> &rVariables) const;
+    void exportToHDF5_basic(const QString &rFilePath, const QString &rModelFileName, const QString &rToolName, const QList<QStringList> &rSystemHierarchies, const QStringList &rComponentNames, const QStringList &rPortNames, const QStringList &rVariableNames, const QStringList &rAliasNames, const QList<QVector<double> > &rDataVectors, const QStringList &rUnits, const QStringList &rQuantities) const;
```

Comment:
what functions is this _basic, leftover ?

Reply:
Yes, it was my temporary implementation before creating the static library. I will remove it.